pkgs/tools/typesetting/tex/texlive/tl2nix.sed

```diff
@@ -30,7 +30,11 @@ $a}
   }
 
   # extract hashes of *.tar.xz
-  s/^containerchecksum (.*)/  sha512.run = "\1";/p
+  /^containerchecksum (.*)/{
+    # save in holdspace, print later if hasRunfiles || hasTlpkg
+    s/^containerchecksum (.*)/  sha512.run = "\1";/
```

Comment:
Do we really have situations when `containerchecksum` is set but `hasRunfiles || hasTlpkg` is not satisfied?

Reply:
`containerchecksum` is always set because of the file `tlpkg/tlobj`, which we ignore, so there are packages doing that (all collections and a handful of doc only packages).